process/block/preprocess/transactions.go

```diff
@@ -1351,13 +1351,12 @@ func (txs *transactions) ProcessMiniBlock(
 
 	defer func() {
 		if err != nil {
-			if scheduledMode {
-				txs.gasHandler.RemoveGasProvidedAsScheduled(processedTxHashes)
-			} else {
-				txs.gasHandler.RemoveGasProvided(processedTxHashes)
-				txs.gasHandler.RemoveGasRefunded(processedTxHashes)
-				txs.gasHandler.RemoveGasPenalized(processedTxHashes)
+			for _, hash := range processedTxHashes {
+				//TODO: Remove it or set log level to Trace
```

Comment:
resolve TODO?

Reply:
We are hunting a rare situation, so I will keep this log on Trace, which anyway should not appear too often, just to catch this error which seems to come from sc execution